ui/src/features/stage/verifications.tsx

```diff
@@ -18,60 +20,108 @@ type Props = {
 export const Verifications = ({ verifications, images }: Props) => {
   const { show } = useModal();
 
+  const [showImplicitVerifications, setShowImplicitVerifications] = useState(false);
+
+  // non-rollout verifications are now included in specs
+  const filteredVerifications = !showImplicitVerifications
+    ? verifications.filter((verfication) => verfication.id !== '')
+    : verifications;
+
+  const hasImplicitVerifications = useMemo(
+    () => verifications.some((v) => v.id === ''),
+    [verifications]
+  );
+
   return (
-    <Table<(typeof verifications)[number]>
-      dataSource={verifications}
-      size='small'
-      pagination={{ hideOnSinglePage: true }}
-      rowKey={(p) => p.id || ''}
-    >
-      <Table.Column<(typeof verifications)[number]>
-        width={28}
-        render={(_, verification) => (
+    <>
+      {hasImplicitVerifications && (
+        <Flex className='mb-4'>
           <Tooltip
-            placement='right'
-            overlay={() => (
-              <div className='p-1'>
-                <div className='font-semibold'>{verification.phase}</div>
-                {verification.message && <div className='mt-1'>{verification.message}</div>}
-              </div>
-            )}
+            title={`Implicit verifications are meta info if stage does not have any explicit verifications defined in spec. Kargo fallback stage's health to how last promotion performed.`}
```

Comment:
@krancour I hope this explains the behaviour of controller.. let me know if you wish to change

Reply:
It's pretty close. Suggested wording change:

```suggestion
            title={`Implicit verifications occur when a stage with no explicit verification process first reaches a healthy state following a promotion.`}
```